src/framework/analyses.ml

```diff
@@ -33,7 +33,7 @@ struct
   let getLocation (n,d) = Node.location n
 
   let pretty_trace () ((n,c) as x) =
-    if get_bool "dbg.trace.context" then dprintf "(%a, %a) on %a \n" Node.pretty_trace n LD.pretty c CilType.Location.pretty (getLocation x)
+    if get_bool "dbg.trace.context" then dprintf "(%a, %a) on %a" Node.pretty_trace n LD.pretty c CilType.Location.pretty (getLocation x)
```

Comment:
correct?

Reply:
Yes, because an individual `pretty`, which is used between other stuff in larger format strings, has no need to unconditionally put a newline after itself. That makes it impossible to include this `pretty` in a single-line output.